test/remote/account_preverified_token_tests.js

```diff
@@ -102,13 +102,6 @@ TestServer.start(config)
         .then(
           function (client) {
             t.ok(client.uid, 'account created')
-            return client.keys()
-          }
-        )
-        .then(
-          function (keys) {
-            t.ok(Buffer.isBuffer(keys.kA), 'kA exists')
-            t.ok(Buffer.isBuffer(keys.wrapKb), 'wrapKb exists')
```

Comment:
I think we should keep the explicit tests for retreiving keys here, since signingup against an unverified account has magic behaviour including re-creating a fresh set of keys.